public/index.html

```diff
@@ -4,6 +4,24 @@
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
     <meta name="theme-color" content="#1400ff">
+    <!-- Primary Meta Tags -->
```

Comment:
Oh btw, did you end up using the tools Natalie sent?
I just took a quick look at one which brought up "near me" when I entered "free food", which now that I think about it does sound like a term worth considering.

Reply:
These are all good questions.

> I wonder about the absence of words like "homeless"/"homelessness", or other terms related to our target demographic.
> I reckon it's not something we want _visibly_ shown to users, but with regards to search, would it potentially be helpful? I'm thinking, for instance, in terms of reaching providers and organizations that might find GoGetta/Streetlives relevant. Which, granted, isn't the primary goal of our SEO...
> Still, any thoughts on this?

Technically, there's no way to to use keywords without them being visible to users. Google now completely ignores any meta keywords for indexing purposes, and relies almost entirely on meta title/description, as well as website content.

In terms of 'homeless', I'd think that this term is relevant, however I'll defer to previous research by the team that suggested that users don't want to associate themselves to this term (I think I read that somewhere)?

>Oh, and what about "Streetlives"?

While I'm not 100% sure how the GoGetta and Streetlives brands co-exisit, I think Streetlives could be included in the landing page copy, potentially at the bottom of the page (i.e. '_GoGetta is an initiative by Streetlives_' or something similar)? It depends how how the team wants to brand the GoGetta page, and how prominent the Streetlives brand needs to be?

>And would it make sense to mention anything about how the data is from "people like you", which has always been touted as a major differentiator? I mean, we do say "community-verified", but I don't know if that'll mean anything to users without any context (we don't even mention which community anywhere).

I like the idea of using more colloquial language, esp. if it's been brought up in previous research.

>Oh btw, did you end up using the tools Natalie sent?
I just took a quick look at one which brought up "near me" when I entered "free food", which now that I think about it does sound like a term worth considering.

I tried using a few of them. Without knowing exactly want users are searching for though, it's hard to generalize what terms to include/build related keyword strings with. In terms of the meta title and description, I think having clear text that succinctly describes the tool and what it does (while including some key words we've identified as important) is ideal.

Reply:
OK, all sounds good.

Specifically regarding Streetlives, it's true that we're intentionally separating the brands for the most part. Still, if someone Googles "Streetlives", we definietly want them coming to this site.
Maybe that's the case anyway (if only because it's also on the streetlives.nyc domain), but worth making sure.